### PR TITLE
[Reviewer: Ellie] Cluster manager plugin updates

### PR DESCRIPTION
**Pull request by rkd-msw**

### CODE REVIEW

**Comment by rkd-msw** [comment]
If these look OK, I'll make identical changes to other repos.

File: memento.root/usr/share/clearwater/clearwater-cluster-manager/plugins/memento_memcached_plugin.py [hunk 1]
```diff
@@ -36,6 +36,7 @@
 from metaswitch.clearwater.cluster_manager.plugin_utils import \
     run_command, write_memcached_cluster_settings
 from metaswitch.clearwater.cluster_manager.alarms import issue_alarm
+from metaswitch.clearwater.cluster_manager import pdlogs
```
Comment (eleanor-merry) [inline memento.root/usr/share/clearwater/clearwater-cluster-manager/plugins/memento_memcached_plugin.py]: Combine this with the next line?

File: memento.root/usr/share/clearwater/clearwater-cluster-manager/plugins/memento_memcached_plugin.py [hunk 2]
```diff
@@ -54,6 +56,9 @@ def key(self):
     def files(self):
         return ["/etc/clearwater/memento_cluster_settings"]
 
+    def cluster_description(self):
+        return "local memcached cluster"
```
Comment (eleanor-merry) [inline memento.root/usr/share/clearwater/clearwater-cluster-manager/plugins/memento_memcached_plugin.py]: Would it be useful to include Memento here?

**Comment by eleanor-merry** [comment]
Looks good

**Pull request merged**